docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -183,3 +183,15 @@ Additionally, the following resources have been removed from the `common` theme:
 - `node_modules/jquery`
 
 If you previously used any of the removed resources in your theme, make sure to add them to your own theme resources instead.
+
+= Support for legacy `redirect_uri` parameter and SPI options has been removed
+
+Previous versions of {project_name} had supported automatic logout of the user and redirecting to the application by opening logout endpoint URL such as
+`http(s)://example-host/auth/realms/my-realm-name/protocol/openid-connect/logout?redirect_uri=encodedRedirectUri`. This functionality was deprecated in {project_name} 18 and has been removed in this version in favor of following the OpenID Connect specification.
+
+As part of this change the following related configuration options for the SPI have been removed:
+
+- `--spi-login-protocol-openid-connect-legacy-logout-redirect-uri`
+- `--spi-login-protocol-openid-connect-suppress-logout-confirmation-screen`
+
+If you were still making use these options or the `redirect_uri` paramater for logout you should implement the link:|https://openid.net/specs/openid-connect-rpinitiated-1_0.html[OpenID Connect RP-Initiated Logout specification] instead.
```

Comment:
Fixing a typo, and adding one more sentence to make it hopefully simpler for people to migrate. 

```suggestion
If you were still making use these options or the `redirect_uri` parameter for logout you should implement the link:https://openid.net/specs/openid-connect-rpinitiated-1_0.html[OpenID Connect RP-Initiated Logout specification] instead.
This allows passing in a redirect URL in the `post_logout_redirect_uri` parameter.
```

Reply:
Fixed the typo, as for adding this to the release notes I don't think that is needed. Previous changes to this functionality also have not been added to the release notes, but only the upgrade guide. Adding to much to the release notes just clutters it, and should mostly be reserved for new features.

Reply:
You skipped the `|` I removed, added that as well now.

Reply:
My bad missed that, thanks